### ISSUE TITLE
Updating jsdom dependency in jest-environment-jsdom

### ISSUE THREAD

**Issue by ryubro** [tmpvar/jsdom#2119]
My react-native project uses typescript. After I introduced `jest-environment-jsdom` the `tsc` command complains like:
```
node_modules/@types/node/index.d.ts(134,13): error TS2300: Duplicate identifier 'require'.
node_modules/@types/react-native/index.d.ts(8604,14): error TS2300: Duplicate identifier 'require'.
```

This is due to that the faulty version of `parse5` accidentally included `@types/node` in `dependencies` not in `devDependencies`. Now `parse5` [has fixed the problem](https://github.com/inikulin/parse5/commit/12a4045ba2be2a38a0a47e323e735961e5de5d73#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) on its version 4.

But `jest-environment-jsdom` still uses the faulty version of `parse5` because [it depends on `jsdom@^11.5.1`](https://github.com/facebook/jest/blob/4da1417653a3fe9732dd0bb71168ad99ec9f71c1/packages/jest-environment-jsdom/package.json#L13) which depends on the faulty version of `parse5@^3.0.2`.

The newest [`jsdom@11.6.0` is already depending on `parse5@4.0.0`](https://github.com/tmpvar/jsdom/blob/dc150a182db8cb17a682e21073c43efd9e0ad2a0/package.json#L33) so if it doesn't create a problem, could you update the `jsdom` dependency in the `jest-environment-jsom` from `11.5.1` to `11.6.0`?

So to sum up:
- `jest-environment-json -> jsdom@^11.5.1 -> parse5@^3.0.2` => Problem!
- `jest-environment-json -> jsdom@^11.6.0 -> parse5@^4.0.0` => Problem solved!

Thank you!

### Basic info:

- **Node.js version:** 8.1.4
- **jsdom version:** 11.5.1

### Minimal reproduction case

No such a thing because this is a simple dependency issue.

### How does similar code behave in browsers?

No such a thing because this is a simple dependency issue.

**Comment by Sebmaster** [comment]
1. We do not maintain `jest-environment-jsdom`.
2. From the dependency graph it looks like this update should happen automatically if you blow away your node_modules and reinstall (or update).

**Comment by Zirro** [comment]
Just to clarify since you mentioned it in the issue on the other repo - jsdom itself is not maintained by the Jest team. We are separate projects.

**Comment by ryubro** [comment]
> 2. From the dependency graph it looks like this update should happen automatically if you blow away your node_modules and reinstall (or update).

Indeed. I was confused with `^` and `~`. After I did `yarn upgrade <the_package>` TSC doesn't complain anymore. Thanks.
